### DISCUSSION TITLE
Feature: Post as a different logged in user.

### DISCUSSION THREAD

**Discussion by bizxta** [Docile-Alligator/Infinity-For-Reddit#932] [Completed Ideas] [answered]
Hello, I'm logged into multiple accounts on the app, but usually only scroll through the app using my main account. Quite often though, I see a post or comment where I want to comment using another account I own. It would be nice have an option in the comment page where I can select which account I want to post as. Happy to expand if you like.

**Comment by Docile-Alligator** [comment]
Nice suggestion.

**Comment by nikhilCad** [comment]
It would be very useful to switch accounts while posting a link to subreddits

**Comment by Wladefant** [comment]
Duplicate: #978

**Comment by gbakeman** [comment, marked as answer]
I think this idea is implemented at this point, unless I'm misunderstanding?

**Reply by bizxta** [reply]
So it is! I did not realise. Worth noting the keyboard pops up when you select the user, which hides the list that pops up. Would be great to have that fixed.

**Reply by gbakeman** [reply]
Ah, very good point - I forgot about the keyboard thing!

**Reply by Wladefant** [reply]
Correct @bizxta please open a bug report about the keyboard issue

**Reply by bizxta** [reply]
Thanks! Done in #998

**Reply by Wladefant** [reply]
@bizxta Thank you!!

**Reply by patrickdemers6** [reply]
Keyboard issue addressed in #1231.
